local-proxy/src/server.ts

```diff
@@ -192,7 +195,20 @@ router.post('/invoke', async (ctx) => {
   }
 });
 
-const db = new DBHandler(resolvePath(tmpDir, 'db'));
+const dbPath = resolvePath(tmpDir, 'db');
+const initDataPath = resolvePath(rootPath, 'template_init_data.json');
```

Comment:
not for this PR: the name of the file should probably come from lycan interface so there's only one source of truth

Reply:
Do you think the source of truth should be top-down server->client, because thinking about it now it will be easier to reverse and export it from the client?